backend/src/LeanCode.AppRating/LeanCode.AppRating.csproj

```diff
@@ -1,10 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
```

Comment:
Do we need that?

Reply:
I needed that so that I could check the type internals, but I guess we don't need that in repository